modules/developer_manual/pages/webdav_api/trashbin.adoc

```diff
@@ -0,0 +1,230 @@
+= Trashbin API
+:toc: right
+:toclevels: 1
+:request-base-path: remote.php/dav/trash-bin
+
+== Introduction
+
+The trash bin API provides support for managing files in ownCloud’s trash bin.
+Specifically, it provides the following functionality:
+
+* xref:list-files[List files in the trash bin]
+* xref:delete-files[Delete files permanently from the trash bin]
+* xref:restore-deleted-file[Restore deleted files to the trash bin]
+
+== List Files
+
+List all files in the trash bin
+
+* Path: `{request-base-path}/<username>`
+* Method: `PROPFIND`
+
+=== Request Parameters
+
+[cols=",,",options="header",]
+|===
+|Attribute 
+|Type 
+|Description
+
+|`username` 
+|string 
+|The username of the user to list files for.
+|===
+
+=== Request Headers
+
+[cols="25%,25%,50%",options="header",]
+|===
+|Header
+|Type
+|Description
+
+|`Depth`
+|int
+a|The maximum level to descend to, when retrieving a listing of the files in the user's trash bin.
```

Comment:
I still see ```a|``` ...

Reply:
Yes, you do :-) That's so that the cell content will format correctly.